tools/requirements.txt

```diff
@@ -0,0 +1,14 @@
+boto3==1.16.24
```

Comment:
shouldn't this file be versionless?
Also, why are is number of packages smaller than the number of packages in requirements-freeze.txt

Reply:
> shouldn't this file be versionless?

```requirements.txt``` is used just once, for the fist time when we merge this code. I kept things simple to use the latest version for these packages. Otherwise I would have to run pip install for these packages on host and then create ```requirements-freeze.txt```

Also per my understanding requirements.txt are generally meant to be locked version, but that not a hard rule.  


>Also, why are is number of packages smaller than the number of packages in requirements-freeze.txt

This is because a single pip vended packages can bring multiple other dependant packages.
 Eg  ```python -m pip install aws-shell==0.2.2``` brings following dependencies.
```
aws-shell==0.2.2
awscli==1.18.190
boto3==1.16.30
botocore==1.19.30
colorama==0.4.3
configobj==5.0.6
docutils==0.15.2
jmespath==0.10.0
prompt-toolkit==1.0.18
pyasn1==0.4.8
Pygments==2.7.3
python-dateutil==2.8.1
PyYAML==5.3.1
rsa==4.5
s3transfer==0.3.3
six==1.15.0
urllib3==1.26.2
wcwidth==0.2.5
```

Reply:
i see